client-libs/typescript/src/openai/mergeChunks.ts

```diff
@@ -1,8 +1,9 @@
 import { ChatCompletion, ChatCompletionChunk } from "openai-beta/resources/chat";
+import { omit } from "lodash-es";
```

Comment:
Ouch, sorry about that. I had a @ts-expect-error for another problem on the relevant line that was masking the missing import!